manager/eris-mint/evm/opcodes/opcodes.go

```diff
@@ -1,4 +1,4 @@
-package vm
+package opcodes
```

Comment:
It is convenient to be able to include the opcodes independent of the entire EVM